libs/payments/ui/src/lib/nestapp/config.ts

```diff
@@ -53,9 +53,4 @@ export class RootConfig {
   @ValidateNested()
   @IsDefined()
   public readonly statsDConfig!: Partial<StatsDConfig>;
-
-  @Type(() => PaymentsGleanConfig)
```

Comment:
Temporarily removing this, which should cause the `nestapp` to fail to startup successfully, and then cause CI to fail.

Once validated, this should be added back.